sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -750,15 +752,15 @@ def list_properties_of_certificate_versions(
             attributes = None
         bundle = await self._client.merge_certificate(
             vault_base_url=self.vault_url,
-            certificate_name=name,
+            certificate_name=certificate_name,
             x509_certificates=x509_certificates,
             certificate_attributes=attributes,
             **kwargs
         )
         return KeyVaultCertificate._from_certificate_bundle(certificate_bundle=bundle)
 
     @distributed_trace_async
-    async def get_issuer(self, name: str, **kwargs: "**Any") -> CertificateIssuer:
+    async def get_issuer(self, issuer_name: str, **kwargs: "**Any") -> CertificateIssuer:
```

Comment:
This method's docstring still uses `name`.

Reply:
thanks so much!